src/Utilities/NumberUtil.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * A class of utilities for dealing with numbers.
+ */
+
+namespace Automattic\WooCommerce\Utilities;
+
+/**
+ * A class of utilities for dealing with numbers.
+ */
+final class NumberUtil {
```

Comment:
Why not just `Number` since it's under the `Utilities` namespace?

Reply:
I think we already discussed this at some point. It's better to have explicit class names for readability, and the name `Number` seems to suggest that the class represents a number, which isn't the case. Also `StringUtil` already exists, and in that particular case `String` couldn't be used because it's a PHP reserved word.